electron-app/package-lock.json

```diff
@@ -11325,19 +11342,22 @@
           "version": "3.0.4",
           "bundled": true,
           "dev": true,
+          "optional": true,
           "requires": {
             "brace-expansion": "^1.1.7"
           }
         },
         "minimist": {
           "version": "0.0.8",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
Hey @isidorosp thanks for the PR! I don't know if it's the fact that you added this `optional: true` to some of the package-lock.json pakages, but something in this PR breaks the app.

I could not run it, I got an electron crash after:

1. cloning this branch
2. Doing `npm ci`
3. Then doing `npm run electron:serve`

Reply:
I did a full wipe today and managed to build, I also did a test of going to 1.2.0\develop -> building -> wiping node_modules, going back to 1.1.1\persist-window-state and also managed to build again, so I'm not sure what this could be. Are you getting any information on the cause of the crash?

Reply:
No because it was not really a crash. It just can't lunch the backend. Did not test too much because I am still in Paris until Sunday.

Reply:
Let's troubleshoot next week, enjoy Paris!

Reply:
can't see the problem again in my desktop linux.